scss/atoms/typography/_mixins.scss

```diff
@@ -0,0 +1,26 @@
+// ==========================================================================
```

Comment:
> No single line comments end up in output CSS files whereas multiline comments do depending on output style
So no comments at all end up in minified files which is really what matter. We also use the "Expanded" style which outputs both single and multi line comments so there is no difference here either. 

> This is handy for when you want to comment things like functions without it ending up in the output code or when you are generating a theme file and want to curate which comments end up in the compiled CSS file. 

This would be true if our output style accommodated this however it does not. Only the compressed output style accommodates this